app/javascript/dashboard/components/helpCenter/Portal/Switch/Index.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <div class="portal" :class="{ active }">
+    <thumbnail :username="portal.name" variant="square" />
+    <div class="actions-container">
+      <header>
+        <div>
+          <p class="portal-title">{{ portal.name }}</p>
+          <p class="portal-count">
+            {{ portal.article_count }}
+            {{ $t('HELP_CENTER.PORTAL.ARTICLES_LABEL') }}
+          </p>
+        </div>
+        <span v-if="active" class="badge">{{
+          $t('HELP_CENTER.PORTAL.ACTIVE_BADGE')
+        }}</span>
+      </header>
+      <div class="portal-locales">
+        <p class="locale-title">
+          {{ $t('HELP_CENTER.PORTAL.CHOOSE_LOCALE_LABEL') }}
+        </p>
+        <ul>
+          <li v-for="locale in portal.locales" :key="locale.code">
+            <label :for="`locale-${locale.code}`" class="locale-item">
+              <input
+                :id="`locale-${locale.code}`"
+                v-model="selectedLocale"
+                type="radio"
+                name="locale"
+                :value="locale.code"
+              />
+              <div>
+                <p>{{ locale.name }}</p>
+                <span>
+                  {{ locale.article_count }}
+                  {{ $t('HELP_CENTER.PORTAL.ARTICLES_LABEL') }} -
+                  {{ locale.code }}
+                </span>
+              </div>
+            </label>
+          </li>
+          <li>
+            <a>+ {{ $t('HELP_CENTER.PORTAL.ADD_NEW_LOCALE') }}</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import thumbnail from 'dashboard/components/widgets/Thumbnail';
+export default {
+  components: {
+    thumbnail,
+  },
+  props: {
+    portal: {
+      type: Object,
+      default: () => ({}),
+    },
+    active: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  data() {
+    return {
+      selectedLocale: '',
+    };
+  },
+  mounted() {
+    this.selectedLocale = this.portal.locales[0].code;
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.portal {
+  background-color: var(--white);
+  padding: var(--space-one);
+  border: 1px solid var(--color-border-dark);
+  border-radius: var(--border-radius-medium);
+  position: relative;
+  display: flex;
+  margin-bottom: var(--space-one);
+
+  &.active {
+    border: 1px solid var(--w-400);
+  }
+
+  .actions-container {
+    margin-left: var(--space-one);
+    flex-grow: 1;
+
+    header {
+      display: flex;
+      align-items: flex-start;
+      justify-content: space-between;
+      margin-bottom: var(--space-one);
+
+      .badge {
```

Comment:
We can use existing label component with `smooth` here.

Reply:
<img width="192" alt="Screenshot2022-07-18 at 00 45 44@2x" src="https://user-images.githubusercontent.com/15716057/179421512-178266c4-8d1c-43a1-9d6a-ee3d5e3551ff.png">

@nithindavid  I tried using class="badge smooth" but I dont think we have this variant of the badge, hence wrote a custom class for this to match the design.

Reply:
Works for now